src/OidcClient.ts

```diff
@@ -136,6 +137,7 @@ export class OidcClient {
             skipUserInfo,
             nonce,
             disablePKCE: this.settings.disablePKCE,
+            omitScopeWhenRequesting: omitScopeWhenRequesting,
```

Comment:
here we can use a shorthand: `omitScopeWhenRequesting: omitScopeWhenRequesting,` -> `omitScopeWhenRequesting ,`

Reply:
updated